src/fondant/pipeline/lightweight_component.py

```diff
@@ -0,0 +1,42 @@
+import typing as t
+from dataclasses import dataclass
+from functools import wraps
+
+
+@dataclass
+class Image:
+    base_image: str = "fondant:latest"
```

Comment:
We should link this to the installed version of Fondant once we add the CI/CD for this.